test/parallel/test-cluster-worker-isdead.js

```diff
@@ -5,13 +5,15 @@ const assert = require('assert');
 
 if (cluster.isMaster) {
   const worker = cluster.fork();
-  assert.ok(
-    !worker.isDead(),
-    'isDead() should return false right after the worker has been created.');
+  const workerDead = worker.isDead();
+  assert.ok(!workerDead,
+            `isDead() returned ${workerDead}. isDead() should return false ` +
+            'right after the worker has been created.');
 
   worker.on('exit', function() {
-    assert.ok(worker.isDead(),
-              'After an event has been emitted, isDead should return true');
+    assert.ok(workerDead,
```

Comment:
Tests have failed:

```
not ok 282 parallel/test-cluster-worker-isdead
  ---
  duration_ms: 0.460
  severity: fail
  stack: |-
    assert.js:45
      throw new errors.AssertionError({
      ^
    
    AssertionError [ERR_ASSERTION]: isDead() returned false. After an event has been emitted, isDead should return true
        at Worker.<anonymous> (/usr/home/iojs/build/workspace/node-test-commit-freebsd/nodes/freebsd11-x64/test/parallel/test-cluster-worker-isdead.js:14:12)
        at emitTwo (events.js:136:13)
        at Worker.emit (events.js:227:7)
        at ChildProcess.worker.process.once (internal/cluster/master.js:184:12)
        at Object.onceWrapper (events.js:335:30)
        at emitTwo (events.js:136:13)
        at ChildProcess.emit (events.js:227:7)
        at Process.ChildProcess._handle.onexit (internal/child_process.js:210:12)
  ...
ok 283 parallel/test-cluster-worker-kill
```

We cannot use the cached variable here. Please restore this to `worker.isDead()`